wellcomeml/datasets/conll.py

```diff
@@ -0,0 +1,56 @@
+from wellcomeml.datasets.download import check_cache_and_download
+
+import os
+import random
+
+def load_data_spacy(data_path, inc_outside=True):
+
+    # Load data in Spacy format:
+    # X = list of sentences (plural) / documents ['the cat ...', 'some dog...', ...]
+    # Y = list of list of entity tags for each sentence [[{'start': 36, 'end': 46, 'label': 'PERSON'}, {..}, ..], ... ]
+    # inc_outside = False: don't include none-entities in the output
+
+    X = []
+    Y = []
+    with open(data_path) as f:
+        lines = f.read().split('-DOCSTART- -X- O O\n')
+        for line in lines:
```

Comment:
i think these are not lines, the are possibly articles, i suggest you name them accordingly